modules/javafx.graphics/src/main/java/com/sun/glass/ui/headless/NestedRunnableProcessor.java

```diff
@@ -0,0 +1,79 @@
+package com.sun.glass.ui.headless;
+
+import com.sun.glass.ui.Application;
+import java.util.LinkedList;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public class NestedRunnableProcessor implements Runnable {
+
+    private final LinkedList<RunLoopEntry> activeRunLoops = new LinkedList<>();
+
+    private final BlockingQueue<Runnable> runnableQueue = new LinkedBlockingQueue<>();
+
+    @Override
+    public void run() {
+        newRunLoop();
+    }
+
+    void invokeLater(final Runnable r) {
```

Comment:
The following methods are never used:
* `invokeLater`
* `runLater`
* `invokeAndWait`
* `stopProcessing`

Reply:
invokeLater is used from HeadlessApplication
Others are removed now.